docs/sources/flow/setup/configure/configure-linux.md

```diff
@@ -1,13 +1,13 @@
 ---
-description: Learn how to configure Grafana Agent Flow on Linux
-title: Configure Grafana Agent Flow on Linux
+description: Learn how to configure Grafana Agent on Linux
```

Comment:
It seems like some of these pages are losing context about what mode they apply to. Is that intentional? If you're linked to this page directly, the only way to identify that it's for Flow mode is to check the navbar and notice that it's in the flow mode section. Maybe that's ok? 

It looks like it's just the configure folder which doesn't clearly specify flow mode in the title/descriptions anymore.

Reply:
You're too fast on reviews :-) 

I've adjusted this text in the metadata and in the opening sentences. It's a balance between overloading the use of "flow'  and losing context s you've noted. I've been re-reading the content and adding flow where the context is weak.